lamindb_setup/core/_hub_core.py

```diff
@@ -137,10 +138,14 @@ def init_storage(
 
 
 def _init_storage(
-    ssettings: StorageSettings, auto_populate_instance: bool, client: Client
+    client: Client,
+    ssettings: StorageSettings,
+    auto_populate_instance: bool,
+    created_by: str | None = None,
```

Comment:
Can you call this `created_by_uuid` and type it with `UUID` instead of `str`? The conversion via `.hex` should come right before hitting the REST API so the the code internals are all properly typed.

Reply:
Can i leave it `created_by`, but redo the typing? Because i think it is just redundant to do both, i.e. renaming to `created_by_uuid` and typing with `UUID`?

Reply:
Now using `UUID`.